metagpt/tools/ut_writer.py

```diff
@@ -250,21 +255,22 @@ def build_api_doc(self, node: dict, path: str, method: str) -> str:
 
         return doc
 
-    def _store(self, data, base, folder, fname):
+    async def _store(self, data, base, folder, fname):
```

Comment:
Referring to the aread function of the common.py, can I encapsulate an awrite, and is it more convenient to use this function?

Reply:
Replace with common.awrite